### PR TITLE
chore!: rename binary from ecs-preview to copilot

### PR DESCRIPTION
**Pull request by efekarakus**
BREAKING CHANGE: the binary name is now `copilot` instead of `ecs-preview`.

_By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice._

### CODE REVIEW

**Comment by efekarakus** [comment]
@iamhopaul123 updated the remaining files, thanks!

File: buildspec.yml [hunk 1]
```diff
@@ -19,28 +19,28 @@ phases:
...
       - ls -lah ./bin/local
-      - ./bin/local/ecs-preview-amd64 --version
+      - ./bin/local/copilot-amd64 --version
   post_build:
     commands:
-      - VERSION=`./bin/local/ecs-preview-amd64 --version`
+      - VERSION=`./bin/local/copilot-amd64 --version`
       - VERSION=`echo $VERSION | grep -oE "[^ ]+$"`
       - COMMIT_VERSION=`git rev-parse --short HEAD`
       - echo "Creating latest and version-tagged artifacts..."
```
Comment (bvtujo) [inline buildspec.yml:28]: I realize this predates the current PR but do we actually create a "latest" artifact in this buildspec? It looks like we only generate the version-tagged ones.
```suggestion
      - echo "Creating version-tagged artifacts..."
```

Comment (efekarakus) [reply buildspec.yml]: Done!

**Comment by efekarakus** [comment]
> Did this also get all the "suggested actions" as well?

si

**Pull request merged**